src/radical/pilot/agent/executing/popen.py

```diff
@@ -202,6 +202,7 @@ def spawn(self, launcher, cu):
             # Create string for environment variable setting
             env_string = ''
           # env_string += '. %s/env.orig\n'                % self._pwd
+            env_string += 'export RADICAL_BASE_DIR="%s"\n' % self._pwd
```

Comment:
@andre-merzky should we use `RADICAL_BASE` here? (according to [this](https://github.com/radical-cybertools/radical.utils/blob/devel/src/radical/utils/misc.py#L853))

Reply:
Ack, done.